### ISSUE TITLE
[A11y] The "Search for ..." hint should be read after the clear button

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/focus-android#2944]
## Steps to reproduce
1. Activate TalkBack.
2. Type a search term in the URL bar.
3. Swipe right until you reach the search hint, displayed under the URL text box.

### Expected behavior
In URL edit mode, the "Search for ..." hint should be read after the clear button, being in the same view.

### Actual behavior
Currently, the reader jumps out of the edit mode and reads the home screen text in the background, then returns to the "Search for..." hint.

### Device information

* Android device: 
LG Nexus 5 (Android 6.0.1)
Google Pixel (Android P beta)

* Focus version: 6.3
